pkg/cli/mockclients_test.go

```diff
@@ -71,6 +71,14 @@ func (c mockConfigAdminServiceClient) RollbackNetworkChange(ctx context.Context,
 	return response, nil
 }
 
+func (c mockConfigAdminServiceClient) RollbackNewNetworkChange(ctx context.Context, in *admin.RollbackRequest, opts ...grpc.CallOption) (*admin.RollbackResponse, error) {
```

Comment:
isn't this exactly the same as next method up?

Reply:
it is but it's needed for the mock to implement the interface, otherwise it will not be be compliant and this fail the build.